README.md

```diff
@@ -1,34 +1,111 @@
-# Tutorial on Trixi.jl at ICOSAHOM 2021
+# Trixi.jl: High-Order Numerical Simulations in Julia
 
 This is the companion repository for the tutorial on
 [Trixi.jl](https://github.com/trixi-framework/Trixi.jl) at
 [ICOSAHOM 2021](https://www.icosahom2020.org):
 
-**Tutorial 2: Trixi.jl**
-*Venue:* gather.town (link to be added later)</br>
-*Date & time:* Wednesday, 14th July 2021, 6:15pm - 8:15pm (CEST)</br>
-*Chaired by:* Hendrik Ranocha, Michael Schlottke-Lakemper</br>
+**Tutorial 2: Trixi.jl**  
+*Venue:* gather.town (link to be added later)  
+*Date & time:* Wednesday, 14th July 2021, 6:15pm - 8:15pm (CEST)  
+*Chaired by:* Hendrik Ranocha, Michael Schlottke-Lakemper  
```

Comment:
I don't like relying on invisible whitespace to get the correct rendering. For example, it's often useful to remove trailing whitespace automatically, which would break the rendering of this block.

Reply:
OK. We can also use `</br>`, though some markdown previews do not render that correctly.

Reply:
I would mostly care about GitHub, so this should hopefully be fine

Reply:
I'm of course open to better suggestions

Reply:
Ah, it is of course `<br />` and then it works also with the other editors ;-) fixed in 6114ca8.